### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_OperatorTab_ ScreenReader: Narrator does not read any information regarding the blank cell values in the table.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5720]
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11yMAS; #A11yTCS; #MAS4.2.3; #MAS1.3.1; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:** 
Application name: Azure Data Studio (Agent, Create External Data, Notebooks)
 Version: 1.8.0-insider (user setup)
 Electron: 3.1.8
 Node.js: 10.2.0
 V8: 6.6.346.32
NVDA Version: 2019.1
JAWS version:2019
OS: Windows_NT x64 10.0.17763

**Additional Details**
Tool name: NVDA
MAS Reference:MAS1.3.1,MAS4.2.3

**Repro Steps:**  
1.Launch Visual Studio code desktop application.
2. Connect to the Database -> Go to the  SQL server agent tab->Select operator tab
3.Start Screen reader as NVDA.
4.Navigate to table.
5.Observe that Screen reader is unnecessarily announcing column for the table.

**Actual:**
Narrator does not read any information regarding the blank cell values in the table. Narrator is only reading out the information about the cell row and column number not the empty value.

**Similar issue has been found under below pages:-**
1.**CSVPreviewData**- Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button.

**Same issue observed with JAWS.**

**Expected:**
Narrator should read some information about the blank cell values in the table.

**User Impact:**
Screen Reader user may not get information about the blank cell values.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
**MAS1.3.1-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**MAS4.2.3-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={30e4fb40-dcc1-45b4-b714-5e2c28b23ae8}

**Attachment for Reference:**
[5720_[Accessibility]A11y_AzureDataStudio_OperatorTab_ ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3232493/5720_.Accessibility.A11y_AzureDataStudio_OperatorTab_.ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
Should be the same kind of fix as https://github.com/microsoft/azuredatastudio/issues/6152
